scripts/apply-patch

```diff
@@ -50,5 +51,13 @@ function path_components {
   echo "${n}"
 }
 
+GIT_ROOT="$(git rev-parse --show-toplevel)"
+if [[ "$(pwd)" != "${GIT_ROOT}" ]]; then
+  echo "ERROR: Script has not been run from the repository root ('${GIT_ROOT}')."
+  echo "Please change directory there and adjust argument paths to match the new working directory."
```

Comment:
```suggestion
  echo "Please move to the root directory and re-run the script."
```

What if we shorten this a little bit? At least from the feedback, I think the main issue was just running from the wrong folder, maybe adjusting the argument path is implied once we ask the user to change directories.

Reply:
It certainly is implied. Sometimes I find being explicit is necessary, and sometimes it's more confusing. I'm happy with your wording too!